pkg/proxy/routes.go

```diff
@@ -100,10 +100,19 @@ func removeToProxyRoutesIPv6() error {
 }
 
 var (
-	// Routing rule for traffic from proxy.
-	fromProxyRule = route.Rule{
-		Priority: linux_defaults.RulePriorityFromProxyIngress,
-		Mark:     linux_defaults.MagicMarkIsProxy,
+	// Routing rule for traffic from ingress proxy.
+	fromIngressProxyRule = route.Rule{
+		Priority: linux_defaults.RulePriorityFromProxy,
+		Mark:     linux_defaults.MagicMarkIngress,
+		Mask:     linux_defaults.MagicMarkHostMask,
+		Table:    linux_defaults.RouteTableFromProxy,
+		Protocol: linux_defaults.RTProto,
+	}
+
+	//nolint:unused // Routing rule for traffic from egress proxy.
```

Comment:
```suggestion
	//nolint:unused // Routing rule for traffic from egress proxy.
```

huh, why is the `unused` needed? We *are* using it below, no?

Reply:
Because it's in a separate commit. Should I make them together?

Reply:
ah! No, then I think it's ok, but the last commit should remove the `nolint` again?